lesson05/main_test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "testing"
+
+func Test_main(t *testing.T) {
```

Comment:
Main function is usually not tested, as it doesn't return anything. This test doesn't do anything, so should be removed.